themes/default/content/blog/opa-support-for-crossguard/index.md

```diff
@@ -196,7 +196,7 @@ We can now navigate to the URL for our update and see that the policy was enforc
 
 ## Using OPA Rego with Pulumi
 
-OPA-based rules are easy to write for Pulumi. The `input` object represents each individual resource that will be deployed by Pulumi, and the rules are applied independently to each Pulumi resource when it is deployed. The schema for each resource is the same as the properties available when defining the resource as defined in the [Pulumi API Reference]({{< relref "/registry" >}}). Critically, although we highlighted Kubernetes above, Pulumi's resource model spans any cloud or Kubernetes, with providers for more than 50 Cloud and SaaS platforms available. And OPA-based rules can target any or all of these in a single deployment.
+OPA-based rules are easy to write for Pulumi. The `input` object represents each individual resource that will be deployed by Pulumi, and the rules are applied independently to each Pulumi resource when it is deployed. The schema for each resource is the same as the properties available when defining the resource as defined in the [Registry]({{< relref "/registry" >}}). Critically, although we highlighted Kubernetes above, Pulumi's resource model spans any cloud or Kubernetes, with providers for more than 50 Cloud and SaaS platforms available. And OPA-based rules can target any or all of these in a single deployment.
```

Comment:
I think this one suffers from the same issue, no? Since we don't have a global `api-docs` landing page, do we want to continue directing to the `registry` root or point to somewhere that is more identical with the original path?

Reply:
ill go ahead and remove that sentence for now.